drivers/dai/intel/ssp/ssp.c

```diff
@@ -9,6 +9,7 @@
 #include <stdint.h>
 #include <zephyr/spinlock.h>
 #include <zephyr/devicetree.h>
+#include <zephyr/pm/device.h>
```

Comment:
To build successfully, you need additionally:
```
#include <zephyr/pm/device_runtime.h>
```

And the test case needs to be modified with:
```
--- a/tests/boards/intel_adsp/ssp/prj.conf
+++ b/tests/boards/intel_adsp/ssp/prj.conf
@@ -3,3 +3,6 @@ CONFIG_DAI=y
 CONFIG_INTEL_MN=y
 CONFIG_DMA_DW_HOST_MASK=1
 CONFIG_HEAP_MEM_POOL_SIZE=512
+CONFIG_PM=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
```

Not sure how to express the dependency correctly. I didn't find any example in other drivers. @ceolin tips on this? I did now locally:

```
--- a/drivers/dai/intel/ssp/Kconfig.ssp
+++ b/drivers/dai/intel/ssp/Kconfig.ssp
@@ -9,6 +9,9 @@ config DAI_INTEL_SSP
        bool "Intel I2S (SSP) Bus Driver for Dai interface"
        default $(dt_compat_enabled,$(DT_COMPAT_DAI_INTEL_SSP))
        select DMA
+       depends on PM
+       depends on PM_DEVICE
+       depends on PM_DEVICE_RUNTIME
```

Reply:
Thanks for catching the missing include! Got lost in copy-pasting.
We can only depend on PM_DEVICE_RUNTIME, as they depend on each other.